tonic_validate/utils/llm_cache.py

```diff
@@ -0,0 +1,20 @@
+from collections import OrderedDict
+
+
+class LLMCache:
```

Comment:
I don't think we need a new file here. Could just rename the `OpenAILLMCache` since there's no code changes